src/modules/trade/components/TradeWidgetPane.tsx

```diff
@@ -23,6 +30,8 @@ export function TradeWidgetPane(sharedProps: SharedProps) {
 
   const { onNavigate, setCustomHref, customNavLabel, setCustomNavLabel } = useCustomNavigation();
 
+  const isBase = chainId === base.id;
```

Comment:
Think we can use `isBaseChainId` helper from the utils package which will also check if it's the testnet version

Reply:
good call